app/controllers/application_controller.rb

```diff
@@ -1,10 +1,79 @@
+require 'pry'
 require './config/environment'
 
 class ApplicationController < Sinatra::Base
 
   configure do
     set :public_folder, 'public'
     set :views, 'app/views'
+    enable :sessions
+    set :session_secret, "evelynwaugh"
   end
 
-end
\ No newline at end of file
+  helpers do
```

Comment:
you should add a current_user method in your helpers too
your helper methods should look like this. 

```ruby 
def logged_in?
  !!current_user 
end 

def current_user 
  @current_user ||= User.find_by_id(session[:user_id])
end 
```
This will give you access to current_user in the views and your other controllers